bindings/localstorage/localstorage.go

```diff
@@ -0,0 +1,240 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation and Dapr Contributors.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package localstorage
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strconv"
+
+	securejoin "github.com/cyphar/filepath-securejoin"
+	"github.com/dapr/components-contrib/bindings"
+	"github.com/dapr/dapr/pkg/logger"
+	"github.com/google/uuid"
+)
+
+const (
+	fileName = "fileName"
```

Comment:
This name is very similar to a parameter. Can you rename this constant to something like `filenameMetadataKey`?

Reply:
Done.